hieradata_aws/production.yaml

```diff
@@ -119,7 +119,7 @@ govuk::apps::content_publisher::aws_s3_bucket: "govuk-production-content-publish
 govuk::apps::content_publisher::google_tag_manager_auth: "O0DrItJxJeQ5Q2W6YCZzvw"
 govuk::apps::content_publisher::google_tag_manager_id: "GTM-NQXC4TG"
 govuk::apps::content_publisher::google_tag_manager_preview: "env-7"
-govuk::apps::content_store::app_domain: publishing.service.gov.uk
+govuk::apps::content_store::app_domain: production.govuk-internal.digital 
```

Comment:
As the GOVUK_APP_DOMAIN is set for all apps (through `/etc/govuk/env.d`), removing this value from the hieradata should have the same effect. Perhaps do that? Or maybe keep this, and add a comment saying that it can be removed in the future?

Reply:
I like the idea of this being removed... so I'll follow up and do the same in staging and force a change on this PR to remove the line. Thanks for checking and commenting!